weblate/machinery/microsoft.py

```diff
@@ -95,8 +95,7 @@ def map_language_code(self, code):
 
         Remove second part of locale in most of cases.
         """
-        return super().map_language_code(code).replace("_", "-").lower()
-
+        return super().map_language_code(code.replace("_", "-").lower())
```

Comment:
This change breaks handling of `en_devel`. Probably it's better to adjust aliases to deal Weblate language codes.